eth/backend.go

```diff
@@ -259,6 +259,12 @@ func CreateConsensusEngine(ctx *node.ServiceContext, chainConfig *params.ChainCo
 		if chainConfig.Istanbul.Epoch != 0 {
 			config.Istanbul.Epoch = chainConfig.Istanbul.Epoch
 		}
+		if chainConfig.Istanbul.LookbackWindow != 0 {
+			if chainConfig.Istanbul.LookbackWindow > chainConfig.Istanbul.Epoch {
```

Comment:
Should this be `if chainConfig.Istanbul.LookbackWindow >= chainConfig.Istanbul.Epoch`? Since we can't count the first block of the epoch? Or even `if chainConfig.Istanbul.LookbackWindow >= chainConfig.Istanbul.Epoch - 1`, since we don't count the last block eitehr?